src/lib/plugin.js

```diff
@@ -208,7 +209,11 @@ class FiveBellsLedger extends EventEmitter2 {
     }
 
     const reconnect = reconnectCore(() => {
-      return new WebSocket(wsUri)
+      return new WebSocket(wsUri, null, {
+        headers: {
+          authorization: 'Bearer ' + options.authToken
```

Comment:
That change is part of https://github.com/interledgerjs/ilp-plugin-bells/pull/132, which this PR is based on. @dappelt can you weigh in?